2022-03-15 16:22:01,252 INFO    MainThread:1008 [wandb_setup.py:_flush():75] Loading settings from /share/etud/e2008984/.config/wandb/settings
2022-03-15 16:22:01,253 INFO    MainThread:1008 [wandb_setup.py:_flush():75] Loading settings from /share/mastoc/projects/erasmus/dl_historical_images/utils/wandb/settings
2022-03-15 16:22:01,253 INFO    MainThread:1008 [wandb_setup.py:_flush():75] Loading settings from environment variables: {}
2022-03-15 16:22:01,253 INFO    MainThread:1008 [wandb_setup.py:_flush():75] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2022-03-15 16:22:01,253 INFO    MainThread:1008 [wandb_init.py:_log_setup():405] Logging user logs to /share/mastoc/projects/erasmus/dl_historical_images/utils/wandb/run-20220315_162201-3an64ks7/logs/debug.log
2022-03-15 16:22:01,253 INFO    MainThread:1008 [wandb_init.py:_log_setup():406] Logging internal logs to /share/mastoc/projects/erasmus/dl_historical_images/utils/wandb/run-20220315_162201-3an64ks7/logs/debug-internal.log
2022-03-15 16:22:01,253 INFO    MainThread:1008 [wandb_init.py:init():439] calling init triggers
2022-03-15 16:22:01,253 INFO    MainThread:1008 [wandb_init.py:init():442] wandb.init called with sweep_config: {}
config: {}
2022-03-15 16:22:01,254 INFO    MainThread:1008 [wandb_init.py:init():458] re-initializing run, found existing run on stack: cygnmw0d
2022-03-15 16:22:01,257 INFO    MainThread:1008 [wandb_run.py:finish():1539] finishing run vibferreira/master_thesis/cygnmw0d
2022-03-15 16:22:01,258 INFO    MainThread:1008 [jupyter.py:save_history():429] not saving jupyter history
2022-03-15 16:22:01,258 INFO    MainThread:1008 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2022-03-15 16:22:01,258 INFO    MainThread:1008 [wandb_init.py:_jupyter_teardown():337] cleaning up jupyter logic
2022-03-15 16:22:01,258 INFO    MainThread:1008 [wandb_run.py:_atexit_cleanup():1702] got exitcode: 0
2022-03-15 16:22:01,259 INFO    MainThread:1008 [wandb_run.py:_restore():1674] restore
2022-03-15 16:22:01,780 INFO    MainThread:1008 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 709
  total_bytes: 709
}

2022-03-15 16:22:01,895 INFO    MainThread:1008 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 709
  total_bytes: 709
}

2022-03-15 16:22:02,087 INFO    MainThread:1008 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 709
  total_bytes: 709
}

2022-03-15 16:22:02,986 INFO    MainThread:1008 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 709
  total_bytes: 3763
}

2022-03-15 16:22:03,089 INFO    MainThread:1008 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 709
  total_bytes: 3929
}

2022-03-15 16:22:03,191 INFO    MainThread:1008 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 709
  total_bytes: 3929
}

2022-03-15 16:22:03,293 INFO    MainThread:1008 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 3929
  total_bytes: 3929
}

2022-03-15 16:22:03,396 INFO    MainThread:1008 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 3929
  total_bytes: 3929
}

2022-03-15 16:22:03,497 INFO    MainThread:1008 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 3929
  total_bytes: 3929
}

2022-03-15 16:22:03,599 INFO    MainThread:1008 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 3929
  total_bytes: 3929
}

2022-03-15 16:22:03,701 INFO    MainThread:1008 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 3929
  total_bytes: 3929
}

2022-03-15 16:22:03,803 INFO    MainThread:1008 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 3929
  total_bytes: 3929
}

2022-03-15 16:22:04,386 INFO    MainThread:1008 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 3929
  total_bytes: 3929
}

2022-03-15 16:22:04,845 INFO    MainThread:1008 [wandb_run.py:_on_finish():1831] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 3929
  total_bytes: 3929
}
local_info {
}

2022-03-15 16:22:05,993 INFO    MainThread:1008 [wandb_run.py:_footer_history_summary_info():2865] rendering history
2022-03-15 16:22:05,994 INFO    MainThread:1008 [wandb_run.py:_footer_history_summary_info():2894] rendering summary
2022-03-15 16:22:05,998 INFO    MainThread:1008 [wandb_run.py:_footer_sync_info():2822] logging synced files
2022-03-15 16:22:06,005 INFO    MainThread:1008 [wandb_init.py:init():492] starting backend
2022-03-15 16:22:06,006 INFO    MainThread:1008 [backend.py:_multiprocessing_setup():99] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-03-15 16:22:06,014 INFO    MainThread:1008 [backend.py:ensure_launched():219] starting backend process...
2022-03-15 16:22:06,065 INFO    MainThread:1008 [backend.py:ensure_launched():224] started backend process with pid: 2308
2022-03-15 16:22:06,067 INFO    MainThread:1008 [wandb_init.py:init():501] backend started and connected
2022-03-15 16:22:06,083 INFO    MainThread:1008 [wandb_run.py:_label_probe_notebook():947] probe notebook
2022-03-15 16:22:06,084 INFO    MainThread:1008 [wandb_run.py:_label_probe_notebook():957] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2022-03-15 16:22:06,084 INFO    MainThread:1008 [wandb_init.py:init():565] updated telemetry
2022-03-15 16:22:06,086 INFO    MainThread:1008 [wandb_init.py:init():596] communicating run to backend with 30 second timeout
2022-03-15 16:22:07,729 INFO    MainThread:1008 [wandb_run.py:_on_init():1759] communicating current version
2022-03-15 16:22:07,795 INFO    MainThread:1008 [wandb_run.py:_on_init():1763] got version response 
2022-03-15 16:22:07,795 INFO    MainThread:1008 [wandb_init.py:init():625] starting run threads in backend
2022-03-15 16:22:09,391 INFO    MainThread:1008 [wandb_run.py:_console_start():1733] atexit reg
2022-03-15 16:22:09,392 INFO    MainThread:1008 [wandb_run.py:_redirect():1606] redirect: SettingsConsole.WRAP
2022-03-15 16:22:09,392 INFO    MainThread:1008 [wandb_run.py:_redirect():1643] Wrapping output streams.
2022-03-15 16:22:09,394 INFO    MainThread:1008 [wandb_run.py:_redirect():1667] Redirects installed.
2022-03-15 16:22:09,394 INFO    MainThread:1008 [wandb_init.py:init():664] run started, returning control to user process
2022-03-15 16:22:09,398 INFO    MainThread:1008 [wandb_watch.py:watch():47] Watching
2022-03-15 16:22:34,697 INFO    MainThread:1008 [wandb_run.py:_atexit_cleanup():1702] got exitcode: 0
2022-03-15 16:22:34,698 INFO    MainThread:1008 [wandb_run.py:_restore():1674] restore
2022-03-15 16:22:35,527 INFO    MainThread:1008 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 709
  total_bytes: 709
}

2022-03-15 16:22:35,668 INFO    MainThread:1008 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 709
  total_bytes: 709
}

2022-03-15 16:22:36,305 INFO    MainThread:1008 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 709
  total_bytes: 709
}

2022-03-15 16:22:36,757 INFO    MainThread:1008 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 709
  total_bytes: 3754
}

2022-03-15 16:22:36,860 INFO    MainThread:1008 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 709
  total_bytes: 3882
}

2022-03-15 16:22:36,966 INFO    MainThread:1008 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 709
  total_bytes: 3882
}

2022-03-15 16:22:37,068 INFO    MainThread:1008 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 3754
  total_bytes: 3882
}

